### ISSUE TITLE
Problem with Turbolinks

### ISSUE THREAD

**Issue by marcoschicote** [mwanji/jscolor-rails#2]
Hi!
Thanks for developing this gem!
I wanted to ask you if there could be a problem when it is used with Turbolinks. I had to change the jscolor.js file so that it installs when the document is load.
Basically I changed the install function to this:

```
    install : function() {
        jscolor.addEvent(document, 'page:load', jscolor.init);
    },
```

Thanks!

**Comment by mwanji** [comment]
Thanks for pointing this out. After some testing, I think that your solution only works when turbolinks are used and a colorpicker is loaded after a turbolink is followed. I think both the "load" and "page:load" event must be handled.

What do you think?

**Comment by mwanji** [comment]
Could you try the [turbolinks](https://github.com/mwanji/jscolor-rails/tree/turbolinks) branch? If it works for you, I'll merge it into master.

**Comment by marcoschicote** [comment]
I tried it and seems to be working. Thanks!

**Comment by marcoschicote** [comment]
The only thing I would argue about this solution is that you would need to keep track of the change when you incorporate new versions of jscolor. Can we think of another way of doing this that does not involve modifying the js?

**Comment by mwanji** [comment]
Great suggestion. I've merged the turbolinks fix onto master and pulled the change into a separate file. So now jscolor.js requires the orginal jscolor file, which I've renamed to jscolor_vendor.js, and applies the Rails-specific modifications.

**Comment by mwanji** [comment]
Fixed in 1.4.2.1, now available on RubyGems.
